### PR TITLE
Add test info to readme

### PR DESCRIPTION
**Pull request by fulder**
As the tests have grown a bit I decided to add some summarized information about the needed `.cirrus.yml` plugin block as well as the required/optional env params for the installation script to the main repo README.

### CODE REVIEW

**Comment by fulder** [comment]
If everything looks fine here feel free to merge this :)

File: README.md [hunk 1]
```diff
@@ -36,3 +36,33 @@ iocage fetch -P jenkins -g https://github.com/ix-plugin-hub/iocage-plugin-index
...
+
+### Block variables
+*  `<PLUGIN_NAME>` should equal (and be replaced with) the plugin JSON file name (without the `.json` file suffix)
+* The `matrix` -> `freebsd_instance` -> `image_family` should match the plugin manifest FreeBSD version. Alternatively this `matrix` can be a list of `freebsd_instances` with different versions if multiple FreeBSD versions are supported or needed to be tested
+* **required** `env` variables:
+  * `PLUGIN_FILE`: should equal the plugin manifest file and is the one parsed inside the `install_script`
+* **optional** `env` variables:
+  * `ADMIN_UI_USER`: should be used if there is a username/password required to reach the admin UI. Value should have the format: `<username>:<password>`
+  * `FOLLOW_REDIRECTS`: if set to `"false"` will not follow redirects to admin UI and treat a 3XX code as success for the admin UI check
+  * `SKIP_UI_CHECK`: if set to `"true"` will skip the admin portal control entirely
```
Comment (sonicaj) [inline README.md:67]: Can we please improve the text a bit, maybe saying admin portal UI check - i don't think "control" is helpful in assuming that

Comment (fulder) [reply README.md]: Sure thing, don't know why I wrote "control" there to begin with. Should be updated to "admin portal UI check" now instead.

**Review by sonicaj — APPROVED** [review]
Thanks

**Pull request merged**